docs/design/autoware-interfaces/ad-api/list/api/manual/local/operator/heartbeat.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: /api/remote/operator/status
+title: /api/manual/local/operator/heartbeat
```

Comment:
Following the two changes are applied. Is this correct?

- `remote` => `manual/local`
- `status` => `heartbeat`

Reply:
>remote => manual/local

`.../remote/operator/status.md => .../local/operator/heartbeat.md`
It seems like git is misunderstanding the file move.
I did a search and replace so it should be correct.


>status => heartbeat

Yes, This is as intended.